src/platform/plugins/shared/discover/public/application/main/components/top_nav/discover_topnav.tsx

```diff
@@ -248,6 +292,8 @@ export const DiscoverTopNav = ({
           ) : undefined
         }
         onESQLDocsFlyoutVisibilityChanged={onESQLDocsFlyoutVisibilityChanged}
+        draft={draft}
+        onDraftChange={TABS_ENABLED ? onDraftChange : undefined}
```

Comment:
Curious if `onDraftChange` causes issues when tabs are disabled or this is just to make it clear it's unused in that case?

Reply:
Thanks for clarifying!